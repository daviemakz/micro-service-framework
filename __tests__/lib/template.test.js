'use strict';

// Import system components
import CommandTemplate from '../../src/lib/template/command';
import ResponseTemplate from '../../src/lib/template/response';

describe('src/lib/validate', () => {
  describe('CommandTemplate()', () => {
    test('to match snapshot', () => {
      expect(CommandTemplate()).toMatchSnapshot();
    });
  });
  describe('ResponseTemplate()', () => {
    test('to match snapshot', () => {
      expect(ResponseTemplate()).toMatchSnapshot();
    });
  });
});
