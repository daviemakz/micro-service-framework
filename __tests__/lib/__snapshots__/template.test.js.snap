// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/lib/validate CommandTemplate() to match snapshot 1`] = `
Object {
  "body": Object {},
  "destination": undefined,
  "funcName": "",
}
`;

exports[`src/lib/validate ResponseTemplate() to match snapshot 1`] = `
Object {
  "resultBody": Object {
    "errData": Object {},
    "resData": Object {},
  },
  "status": Object {
    "command": Object {
      "code": 100,
      "message": "Command completed successfully",
    },
    "transport": Object {
      "code": 1000,
      "message": "Transport completed successfully",
      "responseSource": "",
    },
  },
}
`;
